source/features/remove-label-with-a-single-click.tsx

```diff
@@ -0,0 +1,84 @@
+import './remove-label-with-a-single-click.css';
+import React from 'dom-chef';
+import XIcon from 'octicon/x.svg';
+import select from 'select-dom';
+import oneTime from 'onetime';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import onReplacedElement from '../helpers/on-replaced-element';
```

Comment:
Could you look into using `selector-observer` like in some other features? It's more reliable than this